src/app/cold-storage/rebalance-to-cold-wallet.ts

```diff
@@ -29,7 +29,7 @@ export const rebalanceToColdWallet = async (): Promise<boolean | ApplicationErro
   const offChainService = LndService()
   if (offChainService instanceof Error) return offChainService
 
-  const displayPriceRatio = await getCurrentPriceAsPriceRatio({
+  const displayPriceRatio = await getCurrentPriceAsWalletPriceRatio({
```

Comment:
this should be `getCurrentPriceAsDisplayPriceRatio`, `getCurrentPriceAsWalletPriceRatio` must only be used in tx related to usd wallets

Reply:
Yup planning to do this in a follow-up PR. This PR is just to establish the new `DisplayPriceRatio` object and separate its implementation for the existing `PriceRatio` one that now becomes `WalletPriceRatio`. So places like this remain functionally unchanged, but will be cleaned up to return the new types in a follow-up PR.